packages/banner/src/Banner.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+// import Icon from '@leafygreen-ui/icon';
+import EditIcon from '@leafygreen-ui/icon/dist/Edit';
+import InfoWithCircleIcon from '@leafygreen-ui/icon/dist/InfoWithCircle';
+import WarningIcon from '@leafygreen-ui/icon/dist/Warning';
+import CheckmarkIcon from '@leafygreen-ui/icon/dist/Checkmark';
+import XIcon from '@leafygreen-ui/icon/dist/X';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const Variant = {
+  Info: 'info',
+  Warning: 'warning',
+  Danger: 'danger',
+  Success: 'success',
+} as const;
+
+type Variant = typeof Variant[keyof typeof Variant];
+
+export { Variant };
+
+const baseBannerStyles = css`
+  position: relative;
+  display: flex;
+  flex-grow: 1;
+  align-items: flex-start;
+  justify-content: space-between;
+  min-height: 40px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 20px;
+  padding-right: 12px;
+  border-top: 1px;
+  border-right: 1px;
+  border-bottom: 1px;
+  border-radius: 6px;
```

Comment:
So correct me if I'm wrong, but we intend to use this banner in full width contexts, I.e. at the top of the page. In that context it doesn't make sense to have rounded corners. so I'm curious how to reconcile this?